### PR TITLE
Merge to Deployment Portage for 3.0.4-3.0.4 release

### PR DESCRIPTION
**Pull request by pengyin-shan**
Prepare for version 3.0.4+portage-3.0.4 release on March 1st

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## [3.0.4+portage-3.0.4] - 2022-02-25
+  - Update translation for some wording
+  
 ## [3.0.4+portage-3.0.3] - 2020-02-24
```
Comment (lagoan) [inline CHANGELOG.md:6]: We need to change this time traveling version as you did with the previous one ;)

Comment (portagenetwork) [reply CHANGELOG.md:6]: Sorry what is the version it supposed to be?

Comment (portagenetwork) [reply CHANGELOG.md:6]: Or I should just keep the 3.0.3 but change the date to today?

**Review by portagenetwork — APPROVED** [review]
This version is for 3.0.4-3.0.4 release

**Pull request merged**